zilliqa/src/crypto.rs

```diff
@@ -219,11 +219,11 @@ impl SecretKey {
         Self::from_bytes(&bytes_vec)
     }
 
-    fn as_bls(&self) -> bls_signatures::PrivateKey {
+    pub fn as_bls(&self) -> bls_signatures::PrivateKey {
         bls_signatures::PrivateKey::new(self.bytes)
     }
 
-    fn as_ecdsa(&self) -> k256::ecdsa::SigningKey {
+    pub fn as_ecdsa(&self) -> k256::ecdsa::SigningKey {
```

Comment:
I was of the view that this method didn't really make sense. A validator's secret key isn't tied to any specific ECDSA key or EVM address that is meaningful.